src/Illuminate/Mail/Mailable.php

```diff
@@ -1009,6 +1009,26 @@ public function when($value, $callback, $default = null)
         return $this;
     }
 
+    /**
+     * Get the Markdown template for the message.
+     *
+     * @return string
```

Comment:
can this be null?

Reply:
You're totally right, thanks for noticing that, @GrahamCampbell!

It is fixed now.